code/scripting/api/objs/loop_brief.cpp

```diff
@@ -11,7 +11,7 @@ ADE_VIRTVAR(Text,
 	nullptr,
 	"The text file of the stage",
```

Comment:
This still says "text file".  Everything else looks good though.

Reply:
Done